show/main.py

```diff
@@ -2157,6 +2157,26 @@ def suppress_pending_fib(db):
     click.echo(state)
 
 
+#
+# 'local-users-passwords-reset' command group ("show local-users-passwords-reset ...")
+#
+@cli.command('local-users-passwords-reset')
+@clicommon.pass_db
+def local_users_passwords_reset(db):
+    """Show local-users-passwords-reset state"""
+
+    feature_table = db.cfgdb.get_entry('LOCAL_USERS_PASSWORDS_RESET', 'global')
+
+    hdrs = ['state']
+    data = []
+
+    for key in hdrs:
+        data.append(feature_table.get(key, '').replace('\\n', '\n'))
```

Comment:
Can you please clarify why do we need replace here? I don't see any other API doing this.

Reply:
I used the same as the Banner 'show' command implementation: sonic-utilities#3021